src/reducers/resources.js

```diff
@@ -163,6 +163,10 @@ const addSubjectToNewState = (state, subject, valueSubjectOfKey) => {
     newSubject.labels = [subjectTemplate.label]
   }
 
+  // Add classes if it doesn't already have.
```

Comment:
"if you read the code out loud and it is the same as what you put in the comment ... then you don't need the comment" -- from Sandi Metz when she did OOP class at Stanford.    But we have plenty of these comments, so no biggie.

Reply:
That's assuming that your future self remembers how read JS code ...